STYLEGUIDE.md

```diff
@@ -1,15 +1,17 @@
 # Style Guide
 
-This is a general style guide that shall govern over the Flame Engine repository. The aim is to keep
-a common stable general environment. This includes high level guidance to help with simple decisions
+This is a general style guide that shall govern over all Flame repositories. The aim is to keep a
+common stable general environment. This includes high level guidance to help with simple decisions
```

Comment:
"common stable general" sounds strange to me :/ what are we trying to convey here?

Reply:
i have no idea what I was thinking hahaha must have been a lapse.

Reply:
AH that was already there haha. that explains why I don't know :P 

improved wording!